src/redash_stmo/settings.py

```diff
@@ -19,3 +19,13 @@
 REMOTE_GROUPS_ALLOWED = set_from_string(
     os.environ.get("REDASH_REMOTE_GROUPS_ALLOWED", "")
 )
+
+# Iodide API key for creating new notebooks
+IODIDE_API_KEY = os.environ.get(
+    "REDASH_IODIDE_API_KEY"
+)
+
+# The Redash API key that Iodide uses
+REDASH_OWN_API_KEY_FOR_IODIDE = os.environ.get(
```

Comment:
Let's call this `IODIDE_DEFAULT_API_KEY`/`REDASH_IODIDE_DEFAULT_API_KEY` for now, but better would be to use the individual user's API key, not a generic one. Please file a ticket to implement this next.